infoenergia_api/contrib/pagination.py

```diff
@@ -73,11 +73,19 @@ async def _pagination_links(self, request, request_id, pagination_list, **kwargs
         url = request.url_for(self.endpoint_name, **kwargs)
         url_cursor = await self._next_cursor(request_id, pagination_list.next_cursor)
 
-        next_page = '{url}?cursor={cursor}&limit={limit}'.format(
-            url=url,
-            cursor=url_cursor,
-            limit=pagination_list.page_size
-        ) if url_cursor else False
+        if 'cch' in self.endpoint_name:
```

Comment:
Not important now, but may be we build `next_page` with the information in request.args and avoid this `if`. I note this as technical debt ;)